common/src/chain/config.rs

```diff
@@ -117,16 +117,65 @@ pub fn create_mainnet() -> ChainConfig {
     }
 }
 
+// TODO: use builder type?
+#[allow(clippy::too_many_arguments)]
+pub fn create_custom(
```

Comment:
Maybe `#[cfg(test)]`?

Reply:
Hiding it behind an attribute was easy but trying to configure that in Cargo.toml was quite a bit harder. There is this [one issue](https://github.com/rust-lang/cargo/issues/2911) still open so I'm not sure if there even exist a straight-forward way to enable one or more features for a crate in `dev-dependencies` while still having the same crate in use. If you know how to do it, please let me know but I'll make this as resolved for now.

Reply:
If this is getting out of control, let's drop, and simply put a TODO on it and we'll figure it out later.

Reply:
I don't think it's too bad right now, you just have to add `#[cfg(feature = "testing)]` when you want to use `create_custom()`. Should I delete it or leave as-is for now?

Reply:
That sounds OK. Keep the `#[cfg(feature = "testing)]` for now.